src/main/java/task/Task.java

```diff
@@ -0,0 +1,46 @@
+package task;
+
+public class Task {
+
+    protected boolean done;
```

Comment:
consider giving a better name to this variable so that it sounds more like a boolean variable.